src/pages/executive-advisors.jsx

```diff
@@ -1,98 +1,52 @@
 // Variables
-// import { colors, prefix } from "./../shared/styles.js";
+import { colors, prefix } from "./../shared/styles.js";
 
 // Components
 
 import Layout from "../components/Layout";
+import CardAdvisors from "../components/CardAdvisors";
 
 const StyledExecutiveAdvisorsPage = styled.div``;
 
-const ExecutiveAdvisorsPage = () => {
-  const nav = [
-    {
-      label: "Home",
-      link: "/",
-    },
-    {
-      label: "About Us",
-      link: "/about",
-    },
-    {
-      label: "Services",
-      link: "/services",
-    },
-    {
-      label: "Executive Advisors",
-      link: "/executive-advisors",
-    },
-    {
-      label: "Contact",
-      link: "/contact",
-    },
-    {
-      label: "Blog",
-      link: "/blog",
-    },
-  ];
-
-  const data = {
-    header: {
-      top: {
-        phone: { label: "Call now", href: "+19542583117", icon: "call" },
-        email: {
-          label: "contact@lec.guru",
-          href: "contact@lec.guru",
-          icon: "envelope",
-        },
-      },
-      brand: {
-        icon: "logo",
-        alt: "Lean Enterprise Consulting",
-        to: "/",
-      },
-      nav: nav,
-    },
-    footer: {
-      nav: { heading: "Pages", nav: nav },
-      social: {
-        heading: "Follow Us",
-        links: [
-          {
-            icon: "facebook",
-            link: "https://www.facebook.com/",
-          },
-          {
-            icon: "linkedin",
-            link: "https://www.linkedin.com/",
-          },
-          {
-            icon: "twitter",
-            link: "https://twitter.com/",
-          },
-          {
-            icon: "instagram",
-            link: "https://www.instagram.com/",
-          },
-        ],
-      },
-      contact: {
-        heading: "Contact Us",
-        address: "Fort Lauderdale, FL. USA",
-        phone: "Phone: +954-258-3117",
-        email: "Email: contact@lec.guru",
-      },
-      copyright:
-        "© 2008 - 2022 Lean Enterprise Consulting, Inc. | All Rights Reserved.",
-    },
-  };
+const ExecutiveAdvisorsPage = ({ data }) => {
+  const { teamImages, allDataJson } = data || {};
+  const { edges } = allDataJson;
+  const { node } = edges[0];
+  const { common, executiveAdisorsPage } = node;
+  const { cards } = executiveAdisorsPage;
+  const { header, footer } = common;
 
+  cards.forEach((item, index) => {
+    const image = getImage(teamImages.edges[index]?.node.childImageSharp);
+    item.image = image;
+    console.log(image);
+  });
   return (
     <StyledExecutiveAdvisorsPage>
-      <Layout header={data.header} footer={data.footer}>
-        hola!!
+      <Layout header={header} footer={footer}>
+        <div className={`${prefix}-teamAdvisors`}>
+          <Container className="pt-auto pt-lg-auto">
+            <Row className="my-4 my-lg-5 ps-5 me-5">
+              {cards.map((advisor, index) => (
+                <Col sm={6} key={index} className="mb-4 mb-lg-5 ">
+                  <CardAdvisors
```

Comment:
Maybe we can deconstruct the object instead of passing each property one by one.

Something similar to this:

<CardAdvisors key={index} {...advisor} />